Logging.sln

```diff
@@ -103,6 +105,7 @@ Global
 		src\Adapters.Tests\CommonTestShared\CommonTestShared.projitems*{96b3afdb-abc4-4df2-a46d-de4286e4bb1f}*SharedItemsImports = 4
 		src\Adapters.Tests\Shared\Adapters.Shared.Tests.projitems*{96b3afdb-abc4-4df2-a46d-de4286e4bb1f}*SharedItemsImports = 4
 		src\Adapters.Tests\TraceListener.Tests.Shared\TraceListener.Net40.Tests.Shared.projitems*{96b3afdb-abc4-4df2-a46d-de4286e4bb1f}*SharedItemsImports = 4
+		src\Adapters\CommonShared\CommonShared.projitems*{a539bb1d-29ff-48e8-8d6e-dfcc543dc2b4}*SharedItemsImports = 4
```

Comment:
Perhaps this is a silly question, but why are there multiple references to src\Adapters\CommonShared\CommonShared.projitems*?